src/modules/coreHttpApi/controllers/AttributesController.ts

```diff
@@ -1,25 +1,58 @@
-import { ConsumptionServices, TransportServices } from "@nmshd/runtime";
+import { ConsumptionServices } from "@nmshd/runtime";
 import { Inject } from "typescript-ioc";
-import { Accept, Context, GET, POST, Path, PathParam, Return, ServiceContext } from "typescript-rest";
+import { Accept, Context, Errors, GET, POST, Path, PathParam, Return, ServiceContext } from "typescript-rest";
 import { Envelope } from "../../../infrastructure";
 import { BaseController } from "../common/BaseController";
 
 @Path("/api/v2/Attributes")
 export class AttributesController extends BaseController {
-    public constructor(
-        @Inject private readonly transportServices: TransportServices,
-        @Inject private readonly consumptionServices: ConsumptionServices
-    ) {
+    public constructor(@Inject private readonly consumptionServices: ConsumptionServices) {
         super();
     }
 
     @POST
     @Accept("application/json")
-    public async createAttribute(request: any): Promise<Return.NewResource<Envelope>> {
-        const selfAddress = (await this.transportServices.account.getIdentityInfo()).value.address;
-        if (request?.content?.owner !== selfAddress) throw new Error("You are not allowed to create an attribute that is not owned by yourself");
```

Comment:
I think to be really backwards compatible this check should be re-added. I mean - if owner if specified AND is set to not the own address we just delete it.